opening_hours/utils/hauki_api_client.py

```diff
@@ -14,18 +21,84 @@
 
 
 class HaukiAPIClient:
-    """Hauki API client, minimal functionality implemented."""
-
-    @staticmethod
-    def build_url(endpoint: Literal["resource", "opening_hours", "date_period"], resource_id: str | None = None) -> str:
+    def __init__(self):
         if not settings.HAUKI_API_URL:
             raise HaukiConfigurationError("HAUKI_API_URL environment variable must to be configured.")
 
-        url = urljoin(settings.HAUKI_API_URL, f"/v1/{endpoint}/")
-        if resource_id:
-            url += f"{resource_id}/"
+    ############
+    # resource #
+    ############
+
+    def get_resources(self, *, hauki_resource_ids: list[int], **kwargs: Any) -> HaukiAPIResourceListResponse:
+        # Prepare the URL
+        url = urljoin(settings.HAUKI_API_URL, "/v1/resource/")
```

Comment:
`urljoin` is a bit overkill here I think, could just use a simple f-string.

Reply:
`urljoin` should be fine here 🤔 
If we use simply join strings we must be able to assume that `HAUKI_API_URL` always or never ends in `/`

Reply:
Urljoin also has some quirks that one might not expect:

```python
urljoin("https://github.com/City-of-Helsinki", "foo")
'https://github.com/foo'
```

Reply:
I think you could just use 

```python
base = str(settings.HAUKI_API_URL).removesuffix('/')
url = f"{base}/v1/resource/"
```